src/client/component/localized_strings.cpp

```diff
@@ -30,6 +30,26 @@ namespace localized_strings
 		}
 	}
 
+	bool add(const std::string& key, const std::string& value)
+	{
+		bool result;
+
+		localized_overrides.access([&](localized_map& map)
+		{
+			if (!map.contains(key))
+			{
+				map[key] = value;
+				result = true;
+				return;
+			}
+
+			// put a warning in the console that the localized string is already existing
+			result = false;
+		});
+
+		return result;
```

Comment:
shouldn't there be a gsl::finally callback here because the result boolean may not be defined yet due to the lambda callback for accessing? not entirely sure if i'm even correct on this